### ISSUE TITLE
Gdrive Bug : Not Copying/Moving Many Files.

### ISSUE THREAD

**Issue by LikerPlayer** [UsergeTeam/Userge#110]
It is showing this error: User Rate Limit Exceeded. Rate of requests for user exceed configured project quota. You may consider re-evaluating expected per-user traffic to the API and adjust project quota limits accordingly. You may monitor aggregate quota usage and adjust limits in the API Console.
Solution: Please Add Limit To Copy/Move 10 Files Per Second. You Can Check More About It Here: https://stackoverflow.com/questions/53652829/user-rate-limit-exceeded-with-google-drive-api/53653232#53653232

**Comment by rking32** [comment]
` User Rate Limit Exceeded` is not a bug.
its just a restriction which is added by google.
<b>750GB per day per user </b>

**Comment by LikerPlayer** [comment]
I'm not talking about daily limit exceed, this is user rate limit exceed which is an actions/requests per second made by users, kindly see the link I have attach.

**Comment by rking32** [comment]
> I'm not talking about daily limit exceed, this is user rate limit exceed which is an actions/requests per second made by users, kindly see the link I have attach.

oh, so you are saying to reduce the copying speed or add timeout withing requests ?

**Comment by LikerPlayer** [comment]
Yes Bro. Add Limit to do request between 8 to 10  per second, not do more requests otherwise when we copy files nearly 200 files it stops sometimes at 130, 140, etc and says error user rate limit exceeded, & All Files Are Nearly 100 Mb, That won't become 750gb🤣

**Comment by rking32** [comment]
@LikerPlayer ok lets try it

**Comment by LikerPlayer** [comment]
When will it be solved bro?! @rking32

**Comment by rking32** [comment]
> When will it be solved bro?!
> & Please Remove the invalid question tag, add this to enhancement. @rking32 

Sorry i forgot that. will try on future commits 😊

**Comment by LikerPlayer** [comment]
Yes It's Working Now, Thanks😊. But why is Userge Only Working in Bot Mode, Not Working With Phone Number, Numbers are automatically getting Banned!?🤔

**Comment by rking32** [comment]
> Yes It's Working Now, Thanksblush. But why is Userge Only Working in Bot Mode, Not Working With Phone Number, Numbers are automatically getting Banned!?thinking

don't use fake numbers

**Comment by LikerPlayer** [comment]
I'm Using United-States Number, Anyways, Thank You So Much😊😊😊
